src/Microsoft.AspNetCore.Blazor/UIEventArgs.cs

```diff
@@ -168,7 +168,7 @@ public class UIDataTransferItem
     /// <summary>
     /// Supplies information about an error event that is being raised.
     /// </summary>
-    public class UIErrorEventArgs : UIProgressEventArgs
```

Comment:
This should have some properties https://developer.mozilla.org/en-US/docs/Web/API/ErrorEvent